.github/actions/verify-token/package.json

```diff
@@ -27,7 +27,7 @@
     "@actions/github": "^5.1.1",
     "@octokit/webhooks-types": "^6.9.0",
     "sigstore": "^1.0.0",
-    "tscommon": "file:../tscommon/tscommon-0.0.0.tgz"
+    "tscommon": "file:../tscommon/tscommon-0.1.0.tgz"
```

Comment:
you'll need to update https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/pre-submit.actions.yml#L26 to iterate thru the tarballs

Reply:
Done.